src/javascript/app/pages/trade/markets.jsx

```diff
@@ -289,6 +314,18 @@ class Markets extends React.Component {
         node.dataset.offsetHeight = node.offsetHeight;
     }
 
+    sortMarkets = (markets) => {
+        const sort_market = {};
+        markets.forEach(([key, obj]) => {
+            if (sort_market[obj.subgroup]){
+                sort_market[obj.subgroup].markets.push({ name: obj.name, key, subgroup_name: obj.subgroup_name });
+            } else {
+                sort_market[obj.subgroup] = { markets: [{ name: obj.name, key, subgroup_name: obj.subgroup_name }] };
+            }
+        });
+        return sort_market;
```

Comment:
The logic looks like it's grouping. Can we rename it to `groupMarkets`?

Reply:
resolved